### PR TITLE
Tadeas new

### PR DESCRIPTION
**Pull request by tadeas-kucera**
Tadeas new rules

### CODE REVIEW

File: certora/specs/Base.spec [hunk 1]
```diff
@@ -78,7 +78,7 @@ methods {
 
     // OperatorsRegistryV1
     function _.reportStoppedValidatorCounts(uint32[], uint256) external => DISPATCHER(true);
```
Comment (Roy-Certora) [inline certora/specs/Base.spec]: If the operators registry contract is removed from the scene, that dispatch summary will cause vacuity. Consider commenting-out the summary or replace it with NONDET.

File: certora/specs/RiverV1.spec
Comment (Roy-Certora) [inline certora/specs/RiverV1.spec:1]: Invariant  totalSupplyBasicIntegrity() follows straight from the formula that calculates the shares. Are you sure that is what you desire to prove?

Comment (Roy-Certora) [inline certora/specs/RiverV1.spec:1]: I didn't understand the implementation of depositAdditivityNoGiftsToEachDeposit.
Could you elaborate?

Comment (tadeas-kucera) [reply certora/specs/RiverV1.spec:1]: Lets assume, recipient has sharesBefore amount of shares. There are three scenarios considered in the rule:
1. Deposit amount1 => recipient gains gain1, so their LSEth balance is sharesBefore + gain1.
2. Deposit amount2 => recipient gains gain2, so their LSEth balance is sharesBefore + gain2.
3. Deposit amount1+amount2 => recipient gains gain3, so their LSEth balance is sharesBefore + gain3.
Then we assert, that:
(sharesBefore + gain1) + (sharesBefore + gain2) <= (sharesBefore + gain3) + sharesBefore
Which is equivalent to:
gain1 + gain2 <= gain3
We do not compare two scenarios, in which we would split the investment amount1+amount2 into two separate investments. Instead we test, that there is zero gift for a deposit by comparing three scenarios (note there is "at initial" for each call).

Passing the assert means there are no gifts to each deposit. Did I describe it ok?

Comment (tadeas-kucera) [reply certora/specs/RiverV1.spec:1]: Yes, this simple rule is pure basic and is here to monitor if the code changes in the future. It's very fast to run. I wrote it at the beginning to know if my setup is ok, but I still keep it for the future changes.

**Pull request closed without merging**